packer/scripts/cassandra/download_install.sh

```diff
@@ -7,5 +7,5 @@ sudo apt-get install -y cloud-utils
 sudo apt-get install -y python-dev python-setuptools python-yaml
 
 # download and uncompress Cassandra
-wget -c https://downloads.datastax.com/ddac/ddac-bin.tar.gz -O - | tar -xz
+mkdir cassandra; wget -c http://apache.spinellicreations.com/cassandra/4.0-alpha4/apache-cassandra-4.0-alpha4-bin.tar.gz -O - | tar -xz -C cassandra --strip-components=1
```

Comment:
Ah nice catch on the mirror, I wasn't sure what to use there.

Yes we have to use 4.0-alpha because 3.x doesn't have `allocate_tokens_for_local_replication_factor` and the procedure to correctly set up the tokens using `allocate_tokens_for_keyspace` is not smooth in automation because you have to start seeds, then create a keyspace w/RF=3, then start the other nodes. Using 4.0-alpha was an easier path, we will bump it to GA when it happens.

Reply:
oh true!